kerl

```diff
@@ -1618,6 +1627,16 @@ do_install() {
     ERL_TOP="$KERL_BUILD_DIR/$build_name/otp_src_$rel"
     cd "$ERL_TOP" || exit_install
     init_install_logfile "$rel" "$build_name"
+
+    prev_build_kernel_release=$(grep <"$ERL_TOP"/"$KERL_CONFIG_STORAGE_FILENAME" -o '^uname -r: \(.*\)$' | sed -n 's|^uname -r: \(.*\)$|\1|p')
```

Comment:
This is now "far" from the place where we put the result of `uname -r` in the file; should we change it to a function, or add a comment where the source is created, for example?

Reply:
Probably should be a function

Reply:
Perfect. I'll change that.

Reply:
6372d76: now `uname -r` (the function) and "uname -r" (the label) are tightly bound to the code that uses them (for creation and validation)